persistence-jdbc/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/persistence/jdbc/JdbcPersistenceComponents.scala

```diff
@@ -42,7 +42,7 @@ trait WriteSideJdbcPersistenceComponents extends WriteSidePersistenceComponents
 trait ReadSideJdbcPersistenceComponents extends ReadSidePersistenceComponents with WriteSideJdbcPersistenceComponents {
 
   lazy val offsetTableConfiguration: OffsetTableConfiguration = new OffsetTableConfiguration(
-    configuration, readSideConfig
+    configuration.underlying, readSideConfig
```

Comment:
We can't remove the dependency on `Configuration` here yet. 
See comment further on about `LagomConfigComponent`